snakemake_wrapper_utils/snakemake.py

```
@@ -44,4 +49,35 @@ def get_format(path):
+        # In the wrapper, one shell command for all move statements:
+        move_cmds = "; ".join(move_files(snakemake, mapping))
+        shell("(main_wrapper_cmd [...]; {move_cmds}) {log}")
+    """
+
+    cmds = []
+    for out_tag, tool_out_name in mapping.items():
+        out_name = snakemake.output.get(out_tag, "")
+        if not out_name:
+            raise KeyError(
+                f"The wrapper requires the named output: {out_tag}. Please provide this named output."
+            )
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)